offline/upgrading.md

```diff
@@ -194,54 +226,73 @@ sudo cp ../<OLD_PACKAGE_DIR>/ansible/inventory/offline/artifacts/admin.conf ansi
 
 Log into the assethost, and verify the 'serve-assets' systemd component is running by looking at `sudo lsof -i -P -n | grep LISTEN`, and checking for `8080`. If it's not:
+
 ```
 sudo service serve-assets start
 ```
 
 ### WORKAROUND: old debian key
+
 All of our debian archives up to version 4.12.0 used a now-outdated debian repository signature. Some modifications are required to be able to install everything properly.
 
 First, gather a copy of the 'setup-offline-sources.yml' file from: https://raw.githubusercontent.com/wireapp/wire-server-deploy/kvm_support/ansible/setup-offline-sources.yml .
+
 ```
 wget https://raw.githubusercontent.com/wireapp/wire-server-deploy/kvm_support/ansible/setup-offline-sources.yml
 ```
+
 copy it into the ansible/ directory:
+
 ```
 cp ansible/setup-offline-sources.yml ansible/setup-offline-sources.yml.backup
 cp setup-offline-sources.yml ansible/
 ```
 
 Open it with your prefered text editor and edit the following:
-* find a big block of comments and uncomment everything in it `- name: trust everything...`
-* after the block you will find `- name: Register offline repo key...`. Comment out that segment (do not comment out the part with `- name: Register offline repo`!)
+
+- find a big block of comments and uncomment everything in it `- name: trust everything...`
+- after the block you will find `- name: Register offline repo key...`. Comment out that segment (do not comment out the part with `- name: Register offline repo`!)
 
 If you are doing anything with kubernetes itsself (unlikely!), disable checking for outdated signatures by editing the following file:
+
 ```
 ansible/roles/external/kubespray/roles/container-engine/docker/tasks/main.yml
 ```
-* comment out the block with -name: ensure docker-ce repository public key is installed...
-* comment out the next block -name: ensure docker-ce repository is enabled
+
+- comment out the block with -name: ensure docker-ce repository public key is installed...
+- comment out the next block -name: ensure docker-ce repository is enabled
 
 Now you are ready to start deploying services.
 
 #### WORKAROUND: dependency
+
 some ubuntu systems do not have GPG by default. wire assumes this is already present. ensure you have gpg installed on all of your nodes before continuing to the next step.
 
 #### Populate the assethost, and prepare to install images from it.
+
+If you think you might run into storage issues. Remove the `/opt/assets/` directory as we will copy over new ones in the following steps.
+
 Since docker is already installed on all nodes that need it, push the new container images to the assethost, and seed all container images:
 
 ```
-d ansible-playbook -i ./ansible/inventory/offline/hosts.ini ansible/setup-offline-sources.yml --tags "debs,containers-helm"
+d ansible-playbook -i ./ansible/inventory/offline/hosts.ini ansible/setup-offline-sources.yml
 d ansible-playbook -i ./ansible/inventory/offline/hosts.ini ansible/seed-offline-docker.yml
```

Comment:
I'm starting to feel more and more like ripping all of the docker out of this document.. because if they follow your instructions as written, they will seed both docker and containerd!